lib/net.c

```diff
@@ -54,6 +52,18 @@ uint8_t default_rss_key[GATEKEEPER_RSS_KEY_LEN] = {
 	0x6a, 0x42, 0xb7, 0x3b, 0xbe, 0xac, 0x01, 0xfa,
```

Comment:
Since we are randomizing these bits, we can drop the initialization of `default_rss_key`.